bids/grabbids/bids_layout.py

```diff
@@ -46,10 +46,27 @@ def get_metadata(self, path, **kwargs):
 
         return merged_param_dict
 
```

Comment:
Oh, one thing I didn't think about: `all` is reserved in Python, and I think the best practice is not to use reserved keywords as argument names. Personally I'm okay with it in this case given that the method is short and there won't be any ambiguity, but if it bothers you, we can always change it to `return_list`. Actually maybe `return_list` is clearer (though longer) in that it's easier to guess what it does. Anyway, your call.